site/_includes/partials/chrome-100-hero-section.njk

```diff
@@ -0,0 +1,48 @@
+{% block content %}
+<section id="back-to-top" class="hero-section">
+  <share-button message="#100CoolWebMoments brought back some memories! Loved going through this timeline of the web. Check it out at https://developer.chrome.com/100">
+    <a class="share-button__link gap-right-200 mobile-share-logo"
+        href="https://twitter.com/share?url={{site.url}}{{ page.url | stripDefaultLocale}}&text=##100CoolWebMoments brought back some memories! Loved going through this timeline of the web. Check it out at https://developer.chrome.com/100"
+        target="_blank" data-label="twitter share" data-category="share" data-action="click">
+      <svg width="18" height="20" viewBox="0 0 18 20" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+        <path fill-rule="evenodd" clip-rule="evenodd" d="M12.97 14.81C13.5 14.31 14.21 14 15 14C16.66 14 18 15.34 18 17C18 18.66 16.66 20 15 20C13.34 20 12 18.66 12 17C12 16.76 12.04 16.53 12.09 16.31L5.04 12.19C4.5 12.69 3.79 13 3 13C1.34 13 0 11.66 0 10C0 8.34 1.34 7 3 7C3.79 7 4.5 7.31 5.04 7.81L12.09 3.7C12.04 3.48 12 3.24 12 3C12 1.34 13.34 0 15 0C16.66 0 18 1.34 18 3C18 4.66 16.66 6 15 6C14.21 6 13.49 5.69 12.96 5.19L5.91 9.3C5.96 9.53 6 9.76 6 10C6 10.24 5.96 10.47 5.91 10.7L12.97 14.81ZM16 3C16 2.45 15.55 2 15 2C14.45 2 14 2.45 14 3C14 3.55 14.45 4 15 4C15.55 4 16 3.55 16 3ZM3 11C2.45 11 2 10.55 2 10C2 9.45 2.45 9 3 9C3.55 9 4 9.45 4 10C4 10.55 3.55 11 3 11ZM14 17C14 17.55 14.45 18 15 18C15.55 18 16 17.55 16 17C16 16.45 15.55 16 15 16C14.45 16 14 16.45 14 17Z" fill="#5F6368"/>
+      </svg>
+    </a>
+  </share-button>
+  <div class="hero-wrapper display-flex direction-column align-center justify-content-center lg:direction-row pad-left-400">
+    <div>
+      {% Img src="image/x1Los57vDga6OEMNi1dIJwZ0qvp2/EAwkX7k7d2W0LgKSbZy1.png", alt="Hero image", class="hero-img",
+          width="800", height="859" %}
+    </div>
+    <div class="hero-info text-align-center">
+      {% Img src="image/x1Los57vDga6OEMNi1dIJwZ0qvp2/OGhcqyf7RviBpDR27WHJ.svg", alt="100 image", class="hero-logo",
+          width="283", height="129" %}
+      <h1 class="hero-heading lg:pad-left-300">cool web moments</h1>
+      <p class="hero-desc lg:pad-left-300">Take a scroll down memory lane... Tell us which moments you loved the most. If we’ve missed anything (and we’re sure we have!), tweet us <a href="https://twitter.com/ChromiumDev" target="_blank" rel="noreferrer noopener">@Chromiumdev</a> with</p>
```

Comment:
This paragraph ends with "with", this does not look correct?

Reply:
Updated!